### ISSUE TITLE
support instances with multiple partitions and disks

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#81]
Originally reported of Google Code with ID 25.

```
It would be great if ganeti support multiple partitions and disks in
instance operations. 

The operations that this could be needed is gnt-instance {add,modify}
gnt-backup {import,export} . 
 
```
Originally added on 2008-02-26 11:24:36 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Hi! Support for different disk (and network) configuration is a target for the 1.3
release. Unfortunately it is not for 1.2 as we are trying to keep major changes away
from the stable branch.

Thanks,

Guido
```
Originally added on 2008-02-26 11:35:01 +0000 UTC.

Added Labels: Type-Enhancement
Removed Labels: Type-Defect
Changed State: Accepted
Added to Milestone: Release1.3

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2008-07-03 18:21:28 +0000 UTC.

Added to Milestone: Release2.0
Removed from Milestone: Release1.3

**Comment by Ganeti-Issues-Migrator** [comment]
```
This was implemented in 2.0, marking as fixed.

iustin
```
Originally added on 2009-08-05 08:55:30 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:07 +0000 UTC.

Changed State: Released
